left_lane_topic: "/lane_planner_left/lower"
right_lane_topic: "/lane_planner_right/higher"
park_topic: "/planning/park_waypoints"
vehicle_cmd_topic: "/vehicle_cmd"
control_flags_topic: "/control_flags"
car_lenght: 2
kp: 0.25
kd: 0.2
ki: 0.0
velocity_limit: 2
treshold: 1.0
target: 1
